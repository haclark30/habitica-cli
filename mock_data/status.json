{
    "data": {
        "profile": {
            "name": "test user"
        },
        "stats": {
            "lvl": 1,
            "class": "Warrior",
            "hp": 10,
            "maxHealth": 100,
            "exp": 100,
            "toNextLevel": 1000,
            "mp": 10,
            "maxMP": 50
        }
    }
}