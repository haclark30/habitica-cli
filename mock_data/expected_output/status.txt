test user
--------------------
Level 1 Warrior
--------------------
HP: 10/100
XP: 100/1000
MP: 10/50
